### PR TITLE
Clarify behavior when SseEvent source onEvent throws and exception

### PR DESCRIPTION
**Pull request by NicoNes**
This PR is my proposal to fix [issue 992](https://github.com/eclipse-ee4j/jaxrs-api/issues/992).
Let me know what you think.

Thanks

### CODE REVIEW

**Review by andymc12 — DISMISSED** [review]
LGTM, thanks @NicoNes!

**Comment by spericas** [comment]
@NicoNes Could you please take a look at my comments? Thanks.

**Comment by andymc12** [comment]
@spericas 
> @NicoNes Could you please take a look at my comments? Thanks.

Could you post a link to your comments?  I don't see any concerns/suggestion comments in the PR or in issue #992.

**Comment by spericas** [comment]
> @spericas
> 
> > @NicoNes Could you please take a look at my comments? Thanks.
> 
> Could you post a link to your comments? I don't see any concerns/suggestion comments in the PR or in issue #992.

Sorry, thought that I had clicked the final button but didn't, now I did.

File: jaxrs-api/src/main/java/jakarta/ws/rs/sse/SseEventSource.java [hunk 1]
```diff
@@ -72,6 +72,11 @@
  * <p>
  * In the case of an error condition response, the <code>Throwable</code> passed to the <code>onError</code> consumer
  * <i>should</i> be a WebApplicationException containing the invalid <code>Response</code> object.
+ * <p>
+ * Note that if, for any of the registered event consumers, invocation of {@link Consumer#accept(Object)
+ * Consumer&lt;InboundSseEvent&gt;#accept(InboundSseEvent)} method throws an exception, this is not an error condition.
+ * Thus <code>onError</code> is not invoked and event processing is not stopped.
+ * User is encouraged to handle exception himself in the event consumer code.
```
[the remark this answers is not in the document]
Comment (NicoNes) [reply jaxrs-api/src/main/java/jakarta/ws/rs/sse/SseEventSource.java:79]: Plural or singular, I don't know. Let me know what you think is the best.

[the remark this answers is not in the document]
Comment (NicoNes) [reply jaxrs-api/src/main/java/jakarta/ws/rs/sse/SseEventSource.java:79]: LGTM . I will make the change.

Thanks

**Review by spericas — APPROVED** [review]

**Review by andymc12 — APPROVED** [review]

**Review by chkal — APPROVED** [review]
LGTM. Thanks.

**Comment by andymc12** [comment]
Merging this as it has been approved by 3 committers and has been open for more than the required two weeks.

**Pull request merged**